packages/integration-tests/utils/generatePage.ts

```diff
@@ -0,0 +1,92 @@
+import { Package } from '@sentry/types';
+import { existsSync, mkdirSync, promises } from 'fs';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import path from 'path';
+import webpack from 'webpack';
+
+import webpackConfig from '../webpack.config';
+
+const PACKAGE_PATH = '../../packages';
+
+/**
+ * Generate webpack aliases based on packages in monorepo
+ * Example of an alias: '@sentry/serverless': 'path/to/sentry-javascript/packages/serverless',
+ */
+async function generateSentryAlias(): Promise<Record<string, string>> {
+  const dirents = (await promises.readdir(PACKAGE_PATH, { withFileTypes: true }))
+    .filter(dirent => dirent.isDirectory())
+    .map(dir => dir.name);
+
+  return Object.fromEntries(
+    await Promise.all(
+      dirents.map(async d => {
+        const packageJSON: Package = JSON.parse(
+          await promises.readFile(path.resolve(PACKAGE_PATH, d, 'package.json'), { encoding: 'utf-8' }),
+        );
+        return [packageJSON['name'], path.resolve(PACKAGE_PATH, d)];
+      }),
+    ),
+  );
+}
+
+export async function generatePage(
+  initialization: string,
+  subject: string,
+  template: string,
+  outPath: string,
+): Promise<void> {
+  const localPath = `${outPath}/dist`;
+  const initializationPath = `${initialization}`;
+  const subjectPath = `${subject}`;
+  const templatePath = `${template}`;
```

Comment:
Why do we have to cast to string here? Would just remove these and change their name in the func args.
```suggestion
```